tests/system-fetcher/tests/handler_test.go

```diff
@@ -687,3 +698,18 @@ func cleanupMockSystems(t *testing.T) {
 	}
 	log.D().Info("Successfully reset mock systems")
 }
+
+func applicationLabels(name string, fromTemplate bool) directorSchema.Labels {
+	labels := directorSchema.Labels{
+		"scenarios":           []interface{}{"DEFAULT"},
```

Comment:
The `DEFAULT` scenario looks a little risky. What will happen when we run the system fetcher and it fetched a few hundred systems and add them all in the DEFAULT scenario. Those will make into all the kyma runtimes in the DEFAULT scenario which is unexpected for the clients.

Is this DEFAULT scenario come from the template processing that we use now and we didn't used before?

Reply:
the "DEFAULT" scenario was always there, we just didn't validate the systems labels before